### ISSUE TITLE
File format

### ISSUE THREAD

**Issue by not7cd** [rszczers/pltak#4]
Format pliku na którym operuje program nie jest oczywisty. Czy program przy zapisie powinien dodać rozszerzenie .csv?

**Comment by rszczers** [comment]
Ano, powinien. Program pracuje na plikach .csv w postaci dokładnie takiej, jakie są dostępne na stronach Ministerstwa Finansów i robi to dość brutalnie, tzn. nie sprawdza struktury i w najgorszym razie zawiesza się. Biję się w pierś i żałuję, że nie dołączyłem żadnej dokumentacji. Brak mi czasu i będzie trzeba i na nią trochę poczekać.

ps. poza suffixem .csv program powinien też afair wyrzucać polskie znaki z nazwy pliku, bo aplikacja kliencka ministerstwa sobie z tym nie radzi.

**Comment by not7cd** [comment]
Aktualnie to co robi ten program jest dosyć dobre. Jest szansa na PR ode mnie gdy zabiorę się za C. Czy powinno się dodawać suffix naiwnie? Wydaje mi się, że powinna istnieć lepsza metoda wbudowana w GTK.
